docs/source/notes/get_start_xpu.rst

```diff
@@ -1,109 +1,96 @@
+
+Software Prerequisite
+---------------------
+
+Visit `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html>`_ for more detailed information regarding:
 
+#. Intel GPU driver installation
+#. Intel support package installation
+#. Environment setup
 
-PyTorch for Intel GPUs is compatible with Intel® Data Center GPU Max Series and only supports OS Linux with release 2.4.
+Installation
+------------
 
-Software Prerequisites
-----------------------
+Binaries
+^^^^^^^^
+
+Platform Linux
+""""""""""""""
 
-As a prerequisite, install the driver and required packages by following the `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html>`_.
 
-Set up Environment
-------------------
+Now we have all the required packages installed and environment activated. Use the following commands to install ``pytorch``, ``torchvision``, ``torchaudio`` on Linux.
 
-Before you begin, you need to set up the environment. This can be done by sourcing the ``setvars.sh`` script provided by the ``intel-for-pytorch-gpu-dev`` and  ``intel-pti-dev`` packages.
+For release wheels
 
 .. code-block::
 
-   source ${ONEAPI_ROOT}/setvars.sh
+    pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/xpu
+
+For nightly wheels
+
+.. code-block::
 
-.. note::
-   The ``ONEAPI_ROOT`` is the folder you installed your ``intel-for-pytorch-gpu-dev`` and  ``intel-pti-dev`` packages. Typically, it is located at ``/opt/intel/oneapi/`` or ``~/intel/oneapi/``.
+    pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/xpu
 
-Build from source
------------------
+Platform Windows
+""""""""""""""""
 
-Now we have all the required packages installed and environment acitvated. Use the following commands to install ``pytorch``, ``torchvision``, ``torchaudio`` by building from source. For more details, refer to official guides in `PyTorch from source <https://github.com/pytorch/pytorch?tab=readme-ov-file#intel-gpu-support>`_, `Vision from source <https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#development-installation>`_ and `Audio from source <https://pytorch.org/audio/main/build.linux.html>`_.
+Now we have all the required packages installed and environment activated. Use the following commands to install ``pytorch`` on Windows, build from source for ``torchvision`` and ``torchaudio``.
+
+For release wheels
+
+.. code-block::
+
+    pip3 install torch --index-url https://download.pytorch.org/whl/xpu
+
+For nightly wheels
```

Comment:
If installation instructions are the same for Linux and Windows, not sure what are the benefit of having two separate sections